risc0/zkvm/src/host/recursion/prove/exec.rs

```diff
@@ -394,6 +394,61 @@ impl<'a> ParallelHandler<'a> {
         }
         self
     }
+
+    fn log(&mut self, msg: &str, args: &[BabyBearElem]) {
```

Comment:
This code is copied from the `MachineContext` version of the logging code -- I needed logging for `ParallelHandler` and this is working for me locally. That said, I haven't convinced myself that this is a good solution globally, so I'm interested in review feedback here.